source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,185 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to update arrays in a document.
+
+To update an array, you need to specify an update and the array elements
+you want to update to an update operation. 
+
+Sample Document
+~~~~~~~~~~~~~~~
+
+The following sections feature examples on how to update this sample
```

Comment:
```suggestion
The following sections feature examples that update this sample
```

Nit: To me, the current phrasing implies that the focus on the page is on updating a document (rather than on teaching readers how to update arrays)